.mage/headers.go

```diff
@@ -35,12 +36,16 @@ type HeaderRule struct {
 	Include     []string `yaml:"include"`
 	Exclude     []string `yaml:"exclude"`
 	Header      string   `yaml:"header"`
-	headerLines [][]byte
+	headerLines []*regexp.Regexp
 	Prefix      string `yaml:"prefix"`
 }
 
 func (r *HeaderRule) split() {
-	r.headerLines = bytes.Split([]byte(strings.TrimSpace(r.Header)), []byte("\n"))
+	lines := bytes.Split([]byte(strings.TrimSpace(r.Header)), []byte("\n"))
+	r.headerLines = make([]*regexp.Regexp, 0, len(lines))
+	for _, line := range lines {
+		r.headerLines = append(r.headerLines, regexp.MustCompile(string(line)))
```

Comment:
We could put `^$` around the line to make sure there are no additional contents.